src/components/UserInputBlock.tsx

```diff
@@ -89,6 +89,26 @@ export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent,
     getPersistedState().catch(() => { })
   }, [])
 
+  const resetPersistedState = async (
+    setFieldError: (field: string, message: string) => void,
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+    setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => Promise<void | FormikErrors<InputFormValues>>
+  ): Promise<void> => {
+    try {
+      if (contentId === undefined || persistor === undefined) {
+        return
+      }
+
+      const newPersistedData: PersistorData = { userResponse: '', responseSubmitted: false }
+      await persistor.put(contentId, JSON.stringify(newPersistedData))
+      setInitialResponse('')
```

Comment:
Clarification question: Is this necessary since we're not using the `onReset` prop?

Reply:
Are you referring to all three lines? Without making the `put` the state previously stored via the `get` is displayed on refresh. I actually do not notice a UI difference if I remove line 104.

Reply:
After further testing across the psets it looks like I do not need to set the response in `resetPersistedState`. `setFieldValue()` clears the input and if a user refreshes after resetting then when the `get` is made the UI would render with the empty response value.

Reply:
Is it worth removing them, then (if for no other reason then so we don't get confused when referencing this code in the future)?

Reply:
Yes. I removed them and I'll push the update soon.